app/jobs/regular/update_private_uploads_acl.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Jobs
+  class UpdatePrivateUploadsAcl < Jobs::Base
+
```

Comment:
It might be smart to add a comment here saying this job only executes when the site setting changes. When I first saw it I freaked out a little bit about how it's running on every upload.